doc/deployment/on_premises.md

```diff
@@ -1,33 +1,181 @@
 # On Premises
 
-Pachyderm is built on [Kubernetes](https://kubernetes.io/) and can be backed by an object store of your choice. As such, Pachyderm can run on any on premise platforms/frameworks that support Kubernetes, a persistent disk/volume, and an object store.
+This document is broken down into the following sections, available at the links below
+
+- [Introduction to on-premises deployments](#introduction) takes you through what you need to know about Kubernetes, persistent volumes, object stores and best practices.  That's this page.
+- [Customizing your Pachyderm deployment for on-premises use](./deploy_custom.html) details the various options of the `pachctl deploy custom ...` command for an on-premises deployment.
+- [Single-node Pachyderm deployment](./single-node.html) is the document you should read when deploying Pachyderm for personal, low-volume usage.
```

Comment:
Single-node here is distinctly different than deploy local, right? Probably worth clarifying. Maybe just add "in a persistent non-dev way" at the end of that bullet to clarify?

Reply:
That's clarified in the actual single-node document, I thought. I specify that `deploy local` is only good for testing and evaluation.